slack-base/src/main/java/com/hubspot/slack/client/models/events/channel/SlackChannelSharedEventIF.java

```diff
@@ -0,0 +1,18 @@
+package com.hubspot.slack.client.models.events.channel;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.events.conversation.SlackConversationEventCore;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
```

Comment:
Please correct me if I'm wrong, but I think we don't need `@JsonNaming(SnakeCaseStrategy.class)` if we have `@JsonProperty("connected_team_id")` set for the field.